src/praisonai/praisonai/bots/_session.py

```
@@ -269,14 +274,47 @@ async def chat(
+            # Build enriched context if platform awareness is enabled
+            origin = None
+            reachable_targets = None
+            
+            if self._inject_session_context:
+                # Detect chat type and build origin
+                from .delivery import detect_chat_type
+                chat_type = detect_chat_type(self._platform, chat_id)
+                origin = Origin(
+                    platform=self._platform,
+                    chat_type=chat_type,
+                    display_name=chat_name or chat_id,
```

Ruff (0.15.17)

[error] Undefined name `chat_name`

(F821)